src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,7 +77,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    System.out.println("test");
```

Comment:
Seems like you forgot to remove some debugging prints, if you do think that the recipient line may be helpful in the future definitely feel free to keep it :)

Reply:
removed print lines